app/models/request.rb

```diff
@@ -52,6 +52,10 @@ def total_items
     request_items.sum { |item| item["quantity"] }
   end
 
+  def formatted_partner_email
```

Comment:
This feels like it more accurately belongs on the `user` model, not the request model? And in the above you'd call `request.partner_user&.formatted_email`.

Reply:
Great call! Just made the update